src/dat/view.cljs

```diff
@@ -266,14 +266,14 @@
 (representation/register-representation
   ::pull-summary-string
   (fn [_ _ pull-data]
-    ;[:span
+    [:span
```

Comment:
@bamarco Actually, this will break things, as there are places where it's assumed this is returning a string directly. Was this causing problems with something?